clang/include/clang/AST/FormatString.h

```diff
@@ -780,6 +791,9 @@ bool parseFormatStringHasFormattingSpecifiers(const char *Begin,
                                               const LangOptions &LO,
                                               const TargetInfo &Target);
 
+ArgType wToArgType(int size, bool fast, ASTContext &C);
+ArgType wToArgTypeUnsigned(int size, bool fast, ASTContext &C);
```

Comment:
```suggestion
ArgType wToArgType(int Size, bool Fast, ASTContext &C);
ArgType wToArgTypeUnsigned(int Size, bool Fast, ASTContext &C);
```
for naming conventions. You should take a pass over your changes and make similar corrections.